_docs/install-and-upgrade.md

```diff
@@ -0,0 +1,236 @@
+to install this package from the AUR. For example, to install it using an AUR
+helper:
+
+```
+pacaur -aS thelounge
+yaourt -aS thelounge
+```
+
+Then enable and start the `thelounge.service` unit using:
+
+```
+systemctl enable --now thelounge.service
+```
+
+The Lounge is now up and running **in private mode** at <http://localhost:9000>.
+
+The package provides both a system and a user service to allow you to run The
+Lounge as your liking. Simply build and install the AUR package, and start the
+service:
+
+- **[System](https://wiki.archlinux.org/index.php/Systemd):**
+  `systemctl start thelounge.service`. The configuration is stored in
+  `/etc/thelounge/config.js`and runs as `thelounge` user.
+- **[User](https://wiki.archlinux.org/index.php/Systemd/User):**
+  `systemctl --user start thelounge.service`. The configuration is stored in
+  your home directory in `~/.thelounge/config.js`.
+
+To configure The Lounge, go to
+[the configuration section](/docs/configuration).
+
+You might want to serve The Lounge behind a reverse proxy such as Nginx. To know
+more about the benefits and steps, follow
+[the guide on reverse proxies](/docs/guides/reverse-proxies).
+
+## From npm releases
+
+Installing the [npm package](https://www.npmjs.com/package/thelounge) directly
+allows you to use stable releases on systems where we do not provide native
+packages.
+
+First, make sure [Node.js](https://nodejs.org/) v6.13.0 or more recent is installed
+with `node --version`, as well as [yarn](https://yarnpkg.com/). If not, follow
+the instructions given on the
+[official documentation](https://nodejs.org/en/download/package-manager/) by
+choosing your distribution in the list.
+
+{: .alert.alert-info role="alert"}
+If you are not able to install yarn and need to use [npm](https://npmjs.com), be sure to add the `--unsafe-perm` flag when running `npm install --global --unsafe-perm thelounge`.
+
+Then install The Lounge using:
+
+```
+yarn global add thelounge
+```
+
+{: .alert.alert-info role="alert"}
+Note that installing from npm does not daemonize nor autostart The Lounge.
+
+Start the server manually using:
+
+```
+thelounge start
+```
+
+{: .alert.alert-info role="alert"}
+If `thelounge` is not found by your shell, you will need to add the output of `yarn global bin` to your PATH.
+
+The Lounge is now up and running **in private mode** at <http://localhost:9000>.
+
+Read more about how to use The Lounge from the command line in
+[the CLI usage section](/docs/usage).
+
+Its configuration file is located at `~/.thelounge/config.js`. To configure The
+Lounge, go to [the configuration section](/docs/configuration).
+
+To upgrade The Lounge, simply re-run the `install` command above, and restart it.
+
+You might want to serve The Lounge behind a reverse proxy such as Nginx. To know
+more about the benefits and steps, follow
+[the guide on reverse proxies](/docs/guides/reverse-proxies).
+
+## Docker
```

Comment:
I'm sure you're aware of this but flagging anyway - this section is outdated now

Reply:
Yeah this PR was split up into multiples ones, so when this is rebased, this change will be gone.

Updated documentation is already on master.